lib/bpf/bpf_test.go

```diff
@@ -436,6 +548,107 @@ func TestRootBPFCounter(t *testing.T) {
 	counter.Close()
 }
 
+func TestBPF_udpEvents(t *testing.T) {
```

Comment:
@jakule, I noticed that all other tests are currently skipped. It's not clear to me what the flakiness is. Should I skip this one too?